app/controllers/idv/image_uploads_controller.rb

```diff
@@ -37,12 +35,8 @@ def liveness_checking_enabled?
       IdentityConfig.store.doc_auth_selfie_capture_enabled
     end
 
-    def ial_context
-      @ial_context ||= IalContext.new(
-        ial: sp_session_ial,
-        service_provider: current_sp,
-        user: current_user,
-      )
+    def liveness_checking_required?
+      decorated_sp_session.selfie_required?
```

Comment:
This does not consider config following?

```ruby
return false if Identity::Hostdata.env == 'prod'
return false unless IdentityConfig.store.doc_auth_selfie_capture_enabled
```

Reply:
Oh, i see, it's in the form. But should we make decision in one place?

Reply:
fair point 🤔 ...

consolidated 😄

Reply:
If we put all the checks in `decorated_sp_session.selfie_required?`, then we don't have to duplicate this method here.